### PR TITLE
Update gradle and docs

### PR DESCRIPTION
**Pull request by afischerdev**
Some updates for gradle and the revisions doc.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.2.2'
+        classpath 'com.android.tools.build:gradle:8.5.0'
```
Comment (devemux86) [inline build.gradle:9]: We can use the `8.5.2` version.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-all.zip
```
Comment (devemux86) [inline gradle/wrapper/gradle-wrapper.properties:6]: Don't know if it is better to use the `-bin` distribution?

File: build.gradle (hunk 1 above)
Comment (afischerdev) [reply build.gradle:9]: Ok

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (afischerdev) [reply gradle/wrapper/gradle-wrapper.properties:6]: I don't mind. -bin is smaller.

**Comment by afischerdev** [comment]
Find a minor issue during my testing:
The app doesn't have a profile list after the first start and the visit on the InstallerActivity.

**Pull request merged**